extensions/http-curl/tests/C2VerifyResourceConsumptionInHeartbeat.cpp

```diff
@@ -0,0 +1,133 @@
+/**
```

Comment:
we will have to decide what to do with `ProcessMetrics.h` which creates a `metrics.RuntimeMetrics.ProcessMetrics` node, populating it with some questionable data (like involuntary context switch count)

Reply:
yeah, I am not sure anyone uses those nodes and they hardly contain any useful information (almost all info has been ported to DeviceInfoNode and AgentInformation)
I created https://issues.apache.org/jira/browse/MINIFICPP-1536 to handle these